launch/dynamo-run/src/main.rs

```diff
@@ -186,38 +197,36 @@ async fn wrapper(runtime: dynamo_runtime::Runtime) -> anyhow::Result<()> {
     .await
 }
 
+fn print_help() {
+    println!("{USAGE}");
+    println!("{HELP}");
+    println!(
+        "Available engines: {}",
+        Output::available_engines().join(", ")
+    );
+}
+
+/// Returns a string describing the acceleration status (CUDA/Metal/Vulkan or CPU)
 /// If the user will benefit from CUDA/Metal/Vulkan, remind them to build with it.
 /// If they have it, celebrate!
 // Only mistralrs and llamacpp need to be built with CUDA.
 // The Python engines only need it at runtime.
 #[cfg(any(feature = "mistralrs", feature = "llamacpp"))]
-fn print_cuda(output: &Output) {
-    // These engines maybe be compiled in, but are they the chosen one?
```

Comment:
We do need this part. They could *compile* with `mistralrs` engine, but be running the `vllm` engine. That decides at runtime whether to use CUDA or not. We don't want to print `accelerator=CPU` when vllm will in fact use CUDA.